pages/advanced/performance.es-ES.mdx

```diff
@@ -0,0 +1,122 @@
+# Rendimiento
+
+SWR proporciona una funcionalidad crítica en todo tipo de aplicaciónes web, por lo que el **rendimiento**
+es una prioridad absoluta.
+
+SWR’s tiene **caché** incorporado y la **[deduplicación](/advanced/performance#deduplication)** evitan las solicitudes de red innecesarias,
+pero el rendimiento del propio hook `useSWR` sigue siendo importante. En una aplicación compleja, puede haber cientos de llamadas `useSWR`
+en una solo página.
+
+SWR asegura que su aplicación tiene:
+
+- _no hay peticiones de red innecesarias_
+- _no hay renderizado innecesarios_
+- _no se importa código innecesario_
+
+sin ningún cambio de código por su parte.
+
+## Deduplicación
+
+Es muy común reutilizar los hooks SWR en tu aplicación. Por ejemplo, una aplicación que muestra el avatar del usuario
+actual 5 veces:
+
+```jsx
+
+function useUser() {
+   return useSWR('/api/user', fetcher)
+}
+
+function Avatar () {
+   const { data, error } = useUser()
+   if (error) return <Error />
+   if (!data) return <Spinner />
+
+   return <img src={data.avatar_url} />
+}
+
+function App() {
+  return <>
+    <Avatar />
+    <Avatar />
+    <Avatar />
+    <Avatar />
+    <Avatar />
+  </>
+}
+
+```
+
+Cada componente `<Avatar/>` tiene un hook `useSWR` en su interior. Dado que tienen el mismo key SWR y 
+que se renderizan casi al mismo tiempo, **sólo se hará 1 solicitud de red**.
+
+Puedes reutilizar tus hooks de datos (como `useUser` en el ejemplo anterior) en todas partes, sin preocuparte por el rendimiento
```

Comment:
Here the informal 'you' is used whereas later in the document the formal 'you' is used. To avoid mixing it up it's better to use the impersonal form:
"Se pueden reutilizar los hooks de datos ... sin preocuparse por el rendimiento"